beast/examples/production_runs_2019/datamodel.py

```diff
@@ -0,0 +1,328 @@
+# (Used if ast_with_positions is True), minimum pixel separation between AST
+# position and catalog star used to determine the AST spatial distribution
+ast_pixel_distribution = 10.0 
+
+# ast_reference_image : string (optional, but required if ast_with_positions
+# is True and no X and Y information  is present in the photometry catalog)	
+# Name of the reference image used by DOLPHOT when running the measured 
+# photometry.	            
+ast_reference_image = None
+
+# ast_coord_boundary : None, or list of two arrays (optional)
+# If supplied, these RA/Dec coordinates will be used to limit the region
+# over which ASTs are generated.  Input should be list of two arrays, the
+# first RA and the second Dec, ordered sequentially around the region
+# (either CW or CCW).
+ast_coord_boundary = None
+
+
+#-------------------------------------------
+#Noise Model Artificial Star Test Parameters
+#-------------------------------------------
+
+# astfile : string
+#   pathname of the AST files (single camera ASTs)
+astfile = 'data/14675_LMC-5665ne-12232.gst.fake.fits'
+
+# ast_colnames : list of strings 
+#   names of columns for filters in the AST catalog (AC)
+ast_colnames = np.array(basefilters)
+
+# noisefile : string
+#   create a name for the noise model
+noisefile = project + '/' + project + '_noisemodel.hd5'
+
+# absflux calibration covariance matrix for HST specific filters (AC)
+absflux_a_matrix = absflux_covmat.hst_frac_matrix(filters)
+
+# Distances: distance to the galaxy [min, max, step] or [fixed number]
+distances = [18.5]
+
+# Distance unit (any length or units.mag)
+distance_unit = units.mag
+
+# velocity of galaxy
+velocity = 262.2 * units.km / units.s # LMC velocity from SIMBAD
+
+################
+
+### Stellar grid definition
+
+# log10(Age) -- [min,max,step] to generate the isochrones in years
+#   example [6.0, 10.13, 1.0]
+logt = [6.0, 10.13, 0.15]
+
+# note: Mass is not sampled, instead the isochrone supplied
+#       mass spacing is used instead
+
+# Metallicity : list of floats
+#   Here: Z == Z_initial, NOT Z(t) surface abundance
+#   PARSECv1.2S accepts values 1.e-4 < Z < 0.06
+#   example z = [0.03, 0.019, 0.008, 0.004]
+#   can they be set as [min, max, step]?
+z = [0.03, 0.019, 0.008, 0.004]
+
+# Isochrone Model Grid
+#   Current Choices: Padova or MIST
+#   PadovaWeb() -- `modeltype` param for iso sets from ezpadova
+#      (choices: parsec12s_r14, parsec12s, 2010, 2008, 2002)
+#   MISTWeb() -- `rotation` param (choices: vvcrit0.0=default, vvcrit0.4)
+#
+# Default: PARSEC+CALIBRI
+oiso = isochrone.PadovaWeb()
+# Alternative: PARSEC1.2S -- old grid parameters
+#oiso = isochrone.PadovaWeb(modeltype='parsec12s', filterPMS=True)
+# Alternative: MIST -- v1, no rotation
+#oiso = isochrone.MISTWeb()
+
+# Stellar Atmospheres library definition
+osl = stellib.Tlusty() + stellib.Kurucz()
+
+################
+
+### Dust extinction grid definition
+extLaw = extinction.Gordon16_RvFALaw()
```

Comment:
I recommend using new dust_extinction package to enable revised RvFA extinction curve model where ALaw = F20 (Fitzpatrick et al. 2020, in prep) and BLaw = Gordon03_SMCBar (Gordon et al. 2003):
`extLaw = extinction.Generalized_RvFALaw(ALaw=extinction.Generalized_DustExt(curve='F20'))`

Reply:
Writing the choices for both ALaw and BLaw explicitly:
`extLaw = extinction.Generalized_RvFALaw(ALaw=extinction.Generalized_DustExt(curve='F20'), BLaw=extinction.Gordon03_SMCBar())`